### PR TITLE
#TP-29 ft(Authentication)  signin form with validation

### PR DESCRIPTION
**Pull request by jeandamascyifuzo**
**What this PR does**
This PR is for SignIn into Phantom with a valid email and password

**How you can test it**
git clone you will have to clone this repo if you have not already
git checkout **ft-signin-to-phantom-TP-29** change the branch
npm install for some new plugins installed for it to run
npm run dev you can use this script to run it locally on dev server
in the browser and **/login** to get login form
you can not login with any other email or password except **email: admin@andela.com pass: test123** and you will redirected to blackPage

**Trello**
https://trello.com/c/axlknOTy/29-sign-in-to-phantom

**screen-shoot**
Screenshot from 2022-02-25 01-37-55

### CODE REVIEW

**Review by sezeranoJchrisostome — DISMISSED** [review]

**Review by wakaflorien — DISMISSED** [review]

**Review by chanceDira — APPROVED** [review]

File: src/components/signIn/LoginForm.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+                                placeholder="Password"
+                                className={
+                                    "lg:w-3/4 w-full lg:ml-16 p-2 text-primary border rounded-md outline-none text-sm transition duration-150 ease-in-out mb-4 md:ml-6"} />
+                            {passwordError && <div className="lg:ml-16 md:ml-7 text-red-600 text-sm">{passwordError}</div>}
+                            <p className="font-sans font-medium lg:ml-16 md:ml-6 justify-center text-blue-500"><a href="">Forget Password?</a></p>
+
+                            <div className="flex items-center mt-6">
+                                <button
+                                    className={
+                                        "bg-blue-500 hover:bg-blue-500 text-white font-bold py-2 px-4 rounded lg:ml-16 md:ml-6 lg:w-3/4 w-full"
```
Comment (sezeranoJchrisostome) [inline src/components/signIn/LoginForm.js:94]: Replace bg-blue-500 to bg-primary-500 or any pallet from 100 - 600 accordingly

File: src/assets/style/LoginForm.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    right: 0;
+    bottom: 0;
+    transform: translate(-50%,-25%); 
+  }
+  .letfSide_back-color{
+    background: linear-gradient(0deg, rgba(26, 40, 82, 0.8), rgba(26, 40, 82, 0.8)), linear-gradient(91.12deg, rgba(62, 71, 98, 0.372) 5.82%, rgba(10, 20, 49, 0) 99.07%);
+    box-shadow: 4px 0px 12px 2px rgba(0, 0, 0, 0.33);
+  }
+  .color-h1{
+    color: #1CA0E3;
```
Comment (renepromesse) [inline src/assets/style/LoginForm.css:15]: please work together with @sezeranoJchrisostome , to fix his PR as soon as possible and then bring the colors he difined into yours. Apply all colors where you have colors

**Review by renepromesse — CHANGES_REQUESTED** [review]
Please @jeandamascyifuzo , make sure your page looks like the design.
- no header
- no scrolling
- fix alignment issues, ( make the form centered)
- fix empty spaces

**Review by chanceDira — DISMISSED** [review]

**Comment by mukunzidd** [comment]
@jeandamascyifuzo 
- Rebase the current changes
- Add the correct Trello Link
- Fix the login functionality by actually protecting routes that need protecting

File: src/functions/RanderMyLocation.js [hunk 3]
```diff
@@ -14,7 +14,6 @@ const RanderMyLocation = () => {
         const radius = e.accuracy;
         const circle = L.circle(e.latlng, radius);
         setBbox(e.bounds.toBBoxString().split(","));
-        console.log(position);
         // setPosition({
         //     "lat": bbox[3],
         //     "lng": bbox[2]
```
Comment (renepromesse) [inline src/functions/RanderMyLocation.js:19]: remove this commented lines

**Review by renepromesse — CHANGES_REQUESTED** [review]
Good job @jeandamascyifuzo , please attend the following reviews:
- squash all the 65 commits into 1

**Pull request merged**